src/plugins/navigation_embeddable/public/components/editor/navigation_embeddable_panel_editor.tsx

```diff
@@ -61,9 +57,11 @@ const NavigationEmbeddablePanelEditor = ({
   initialInput: Partial<NavigationEmbeddableInput>;
   onSave: (input: Partial<NavigationEmbeddableInput>) => void;
 }) => {
-  const isDarkTheme = useObservable(coreServices.theme.theme$)?.darkMode;
   const editLinkFlyoutRef: React.RefObject<HTMLDivElement> = useMemo(() => React.createRef(), []);
 
+  const [currentLayout, setCurrentLayout] = useState<NavigationLayoutType>(
+    initialInput.layout ?? NAV_VERTICAL_LAYOUT
+  );
```

Comment:
Should the `vertical` layout be the default? 🤔 And, if it is the default, should it be the first option in the `EuiButtonGroup` component?

![Screenshot 2023-07-25 at 10 38 20 AM](https://github.com/elastic/kibana/assets/8698078/bd2dcfea-c65e-4fb6-b4cf-b43571f6ad83)